src/main/kotlin/no/nav/syfo/application/ForesporselService.kt

```diff
@@ -8,6 +8,7 @@ import no.nav.syfo.domain.Virksomhetsnummer
 class ForesporselService(
     private val varselProducer: IVarselProducer,
     private val repository: IForesporselRepository,
+    private val journalforingService: IJournalforingService,
 ) {
     fun createForesporsel(
```

Comment:
Synes vi burde journalføre i denne funksjonen i tillegg til å ha en cronjob som plukker opp journalføringer som feiler 😊
- Synes det i koden blir tydeligere at journalføringen henger sammen med at en oppfølgingsplan blir forespurt.
- Vi slipper å gjøre en spørring mot databasen hvert eneste minutt, der 99% av disse spørringene vil være unødvendige

Reply:
Kan prøve ut dette, men vil da gjerne gjøre det i en egen PR sånn at alle kan se hva det innebærer (dette gjelder jo i så fall også alle de andre app'ene der vi journalfører).

Reply:
Ok 😊 Vi må jo ikke skrive alle appene over på denne måten med en gang, men kan jo bestemme oss for å gradvis gå over til dette mønsteret om vi liker det 😊